src/components/ProfilePage.jsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Container, Row, Col } from 'reactstrap';
-
-import ProfileAvatar from './ProfileAvatar';
-import FullName from './FullName';
-import UserLocation from './UserLocation';
-import Education from './Education';
-import SocialLinks from './SocialLinks';
-import Bio from './Bio';
-import MyCertificates from './MyCertificates';
-
-class UserProfile extends React.Component {
+import { connect } from 'react-redux';
+
+// Actions
+import {
+  fetchProfile,
+  saveProfile,
+  saveProfilePhoto,
+  deleteProfilePhoto,
+  openField,
+  closeField,
+} from '../actions/ProfileActions';
```

Comment:
I'm not particularly happy with combining this Page component with the higher order redux-connected component. I see actions imported at the top that are later mapped (with dispatch) to props. It's tempting to just use those imported functions since they're the same name.

I think we would be better served to keep these things separate, though IMO ideally right next to each other in the file structure. Something like: `ProfilePage.jsx` and `ProfilePageConnected.jsx`

Reply:
Yeaaaah, lemme think about that for a sec... it's a good point.  Normally I just universally do `this.props.fetchProfile` (or whatever action) when I need it, which avoids the problem, but I can see how that'd be confusing.  🤔